appinventor/appengine/src/com/google/appinventor/client/explorer/SourceStructureExplorer.java

```diff
@@ -104,7 +110,7 @@ public void onOpen(OpenEvent<TreeItem> event) {
     });
     tree.addSelectionHandler(new SelectionHandler<TreeItem>() {
       @Override
-      public void onSelection(SelectionEvent<TreeItem> event) {
+      public void onSelection(SelectionEvent<TreeItem> event) { 
```

Comment:
Stray space made it in here. This change should be reverted. We try to keep our whitespace diff clean.